staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -171,6 +177,22 @@ var (
 				},
 			},
 		})
+	SuggestedPriorityLevelConfigurationNodeHealth = newPriorityLevelConfiguration(
+		"node-health",
+		flowcontrol.PriorityLevelConfigurationSpec{
+			Type: flowcontrol.PriorityLevelEnablementLimited,
+			Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+				AssuredConcurrencyShares: 100,
```

Comment:
The node health traffic, is it O(number of nodes) in the cluster and each node sends health status to the apiserver on a regular interval? If so, `100` seems a bit high to me especially when we don't have borrowing. 

With `100` added for node health, we are roughly allocating `1/3` of the global concurrency share to node-health traffic. 

Some current stats:
- currently we allocate concurrency share of `30` for all kubelet traffic. 
- `workload-low` has the highest concurrency share of `100`, this is pretty much for all in-cluster service account traffic.

Reply:
> With 100 added for node health, we are roughly allocating 1/3 of the global concurrency share to node-health traffic.

And this 1/3 is roughly what we see in large clusters. Hence the value.
[I will let @mborsz to describe a bit more on the reasoning behind this number.]

If our numbers are visibly different than yours, I'm wondering how should we proceed.
I strongly believe that the rule itself is very important, because health reporting is more important than any other actions done by Kubelets (otherwise nodes may become unready etc.). But I'm wondering how should we solve the value here (I would really like to avoid exposing this as a knob of kube-apiserver).
Any ideas?

Reply:
Sorry, it's a bug (I incorrectly overestimated global concurrency share). Indeed, it should be lower value, like ~30-50.

Reasoning behind this number: In the idle cluster (no pod change), nearly all requests are for node health checking (~1.1K QPS in 5k scale) and we see approx. ~6 cores usage for kube-apiserver, so we can assume that node-health check consumes approx. 10-15% of total master capacity (depending on how we exactly calculate this).

The current global concurrency share is 205, so to get 15% we need 36 shares (36 / (205 + 36). Rounding up, I suggest using 40.